etcdmain/help.go

```diff
@@ -161,7 +161,18 @@ given by the consensus protocol.
 profiling flags:
 	--enable-pprof 'false'
 		Enable runtime profiling data via HTTP server. Address is at client URL + "/debug/pprof/"
+
 	--metrics 'basic'
 	  Set level of detail for exported metrics, specify 'extensive' to include histogram metrics.
+
+auth flags:
+	--auth-token-type 'simple'
```

Comment:
Sorry I misunderstood this comment. The latest PR still has two flags: `--auth-token-type` for specifying type and `--auth-token-opts` for jwt specific options. Should I also unify them into a single one? I prefer the separated style a little bit because the parsing rule is slightly simpler. If we should provide the single big flag, I'll change.